custom_components/tuya_local/devices/loratap_relay.yaml

```diff
@@ -0,0 +1,70 @@
+name: Loratap Relay
+products:
+  - id: owzofmjyg7foteb6
+    name: Loratap RR500W
+primary_entity:
+  entity: switch
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
```

Comment:
I suggest to add cycle_time and random_time (and maybe inching marked as optional) to the primary entity so they are available if wanted.